### PR TITLE
KH-114: Add 'Hb1AC' and 'OGTT' tests to Health Center - NCD Screening form

### PR DESCRIPTION
**Pull request by kazlaw**
https://issues.openmrs.org/browse/KH-114

### CODE REVIEW

File: configs/openmrs_config/concepts/labs.csv [hunk 1]
```diff
@@ -5,6 +5,7 @@ f5f76f5e-2e6a-4112-ab4e-629299829ae3,,,Total Cholestrol,កូលេស្តេ
 c423325e-be95-42e2-ae38-b940c2586327,,,Albumin (Urine),តេស្តអាលប៊ុយម៊ីន (ក្នុងទឹកនោម),Albumin (Urine),តេស្តអាលប៊ុយម៊ីន (ក្នុងទឹកនោម),0,90,mg/g,,,Test,Text,,,,
 07015405-c865-4a22-8ad1-c8863d5f005b,,,Ketones (Urine),ពិនិត្យរកសារជាតិកេតូន (ក្នុងទឹកនោម),Ketones (Urine),ពិនិត្យរកសារជាតិកេតូន (ក្នុងទឹកនោម),0,9,mmol/L,,,Test,Text,,,,
 a8af7520-1350-11df-a1f1-0026b9348838,,,HBA1c,អេម៉ូក្លូប៊ីនអេវ័នស៊ី,HBA1c,អេម៉ូក្លូប៊ីនអេវ័នស៊ី,0,20,%,,,Test,Numeric,,,,
+163594AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,,OGTT,តេស្តកម្រិតជាតិស្ករក្នុងឈាម២ម៉ោងក្រោយផឹកទឹកស្ករ៧៥ក្រាម,OGTT,តេស្តកម្រិតជាតិស្ករក្នុងឈាម២ម៉ោងក្រោយផឹកទឹកស្ករ៧៥ក្រាម,20,450,mg/dl,,,Test,Numeric,,,,
```
Comment (kdaud) [inline configs/openmrs_config/concepts/labs.csv:8]: @kazlaw shouldn't we add HBA1c as well?

Comment (kazlaw) [reply configs/openmrs_config/concepts/labs.csv:8]: @kdaud  HBA1c already exists under this Uuid `a8af7520-1350-11df-a1f1-0026b9348838` in the file

File: configs/openmrs_config/ampathforms/health_center_NCD_Screening.json [hunk 2]
```diff
@@ -328,6 +328,30 @@
...
+                            "type": "obs",
+                            "validators": []
+                        },
+                        {
+                            "id": "OGTT",
+                            "questionOptions": {
+                                "rendering": "number",
+                                "concept": "163594AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                                "max": "450",
+                                "min": "20",
```
Comment (kdaud) [inline configs/openmrs_config/ampathforms/health_center_NCD_Screening.json:350]: I may not really tell what should be the min and max value of the concept. Are these values of OGTT derived from computation of certain patient parameters(data)?

Comment (kazlaw) [reply configs/openmrs_config/ampathforms/health_center_NCD_Screening.json:350]: If you search by the Uuid for that concept in question the values are given.

**Review by kdaud — APPROVED** [review]
LGTM

cc: @nuhuhmutebi

File: configs/openmrs_config/ampathforms/health_center_NCD_Screening.json [hunk 3]
```diff
@@ -328,6 +328,30 @@
                             },
                             "type": "obs",
                             "validators": []
+                        },
+                        {
+                            "id": "HBA1c",
```
Comment (kdaud) [inline configs/openmrs_config/ampathforms/health_center_NCD_Screening.json:333]: @kazlaw I think we were supposed to add `Hb1AC` not `HBA1c` test as per the ticket requirement.

**Pull request merged**